tck/src/main/java/org/eclipse/microprofile/lra/tck/participant/api/ActivityController.java

```diff
@@ -335,6 +339,14 @@ private String restPutInvocation(URL lraURL, String path, String bodyText) {
         return id;
     }
 
+    @PUT
+    @Path(MANDATORY_LRA_RESOURCE_PATH)
+    @LRA(LRA.Type.MANDATORY)
+    public Response activityWithMandatoryLRA(@HeaderParam(LRA_HTTP_RECOVERY_HEADER) String rcvId,
+                                             @HeaderParam(LRA_HTTP_HEADER) String lraId) {
+        return activityWithLRA(rcvId, lraId);
```

Comment:
Change the TCK test such that it subsequently verifies the compensation handler for activityWithMandatoryLRA is called during compensation

Reply:
see the comment below where I added the check to verify that compensate was called. Apologies for squashing but the only change was to ask the LRA aware service how many times compensate had been called before and after the service invocation:
> int[] cnt = {completedCount(true), completedCount(false)};
followed by an assertion that the compensation count had incremented.

Reply:
OK